tools/e2e/e2e.go

```diff
@@ -125,18 +128,20 @@ func Test(conf Config) {
 	}
 	color.Success.Println("updated users, groups, things and channels")
 
-	// Send and Receive messages from channels
+	// Send messages to channels
 	if err := messaging(s, conf, token, things, channels); err != nil {
 		exit(err)
 	}
-	color.Success.Println("sent and received messages from channels")
+	color.Success.Println("sent messages to channels")
 }
 
+// exit exits the program with an error
 func exit(err error) {
 	color.Error.Println(err.Error())
 	os.Exit(1)
 }
 
+// createUser creates user.
```

Comment:
No need for this comment since func is not exported and it is self-documenting. Same for other comments for unexported functions.